resources/views/vendor/feed/rss.blade.php

```diff
@@ -28,7 +28,10 @@
                 @foreach($item->category as $category)
                     <category>{{ $category }}</category>
                 @endforeach
+				@if($item->__isset('enclosure'))
+				<enclosure url="{{ url($item->enclosure) }}" length="{{ $item->enclosureLength }}" type="{{ $item->enclosureType }}" />
+				@endif
             </item>
         @endforeach
     </channel>
-</rss>
+</rss>
```

Comment:
I believe my editor has removed the newline at EOF because [`.editorconfig` specifies `insert_final_newline = false`](https://github.com/cdzombak/Lychee/blob/591b3fab72fba89477263cf3459c215c56a49ffb/.editorconfig#L6).